lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -2182,7 +2247,10 @@ selectTxs = fmap concatUnzip . mapM select . chunksOf chunkSize
             []
 
         pure
-            ( inputs `resolveWith` resolvedInputs
+            ( inputs
+                `resolveInputWith`resolvedInputs
```

Comment:
```suggestion
                `resolveInputWith` resolvedInputs
```

Reply:
Fixed in fd0c08234f222349de62cedcf8a66f841af46c60.